src/node.cc

```diff
@@ -2628,6 +2633,17 @@ static void EnvGetter(Local<Name> property,
 static void EnvSetter(Local<Name> property,
                       Local<Value> value,
                       const PropertyCallbackInfo<Value>& info) {
+  if (config_pending_deprecation && !env_nonstring_deprecation_warned &&
+      !value->IsString()) {
```

Comment:
Can we also allow numbers + booleans, here and in the doc? I don’t see anything wrong with the well-defined behaviour we get from them, plus at least in the case of numbers doing this is pretty common (I count 3 test files in our suite alone).

Reply:
:+1: makes sense to me.